cve_bin_tool/csv2cve.py

```diff
@@ -53,11 +53,12 @@ def csv2cve(filename):
             LOGGER.error("Error: invalid CSV")
             return ERR_BADCSV
 
-        required_columns = ["vendor", "product", "version"]
-        for column in required_columns:
-            if column not in csvdata.fieldnames:
-                LOGGER.error(f"Error: no {column} column found")
-                return ERR_MISSINGCOLUMN
+        required_columns = {"vendor", "product", "version"}
+        csv_columns = set(csvdata.fieldnames)
+        missing_columns = required_columns - csv_columns
+        if missing_columns != set():
+            LOGGER.error(f"Error: no {missing_columns} columns found")
+            return ERR_MISSINGCOLUMN
```

Comment:
I never really cared about it failing out early since you'd have to open the file to fix it and probably notice the problem then, but this is a nice usability improvement!